src/ORM/Marshaller.php

```diff
@@ -289,6 +294,7 @@ public function many(array $data, array $options = [])
     protected function _belongsToMany(Association $assoc, array $data, $options = [])
     {
         $associated = isset($options['associated']) ? $options['associated'] : [];
+        $forceTargetSave = isset($options['forceTargetSave']) ? $options['forceTargetSave'] : false;
```

Comment:
The marshaller doesn't actually save anything. Perhaps this should be `forceNew` ?